app/containers/TabController.js

```diff
@@ -1,13 +1,12 @@
 import React, { PropTypes, Component } from 'react';
-import css from '../styles/tabs.scss';
+import '../styles/tabs.scss';
 
 class TabController extends Component {
     constructor(props) {
         super(props);
         this.state = {
             selectedTabIndex: props.defaultSelectedTabIndex,
         };
-        this.setSelectedTabIndex = this::this.setSelectedTabIndex;
     }
 
     getChildContext() {
```

Comment:
If you mean getChildContext it's actually used by other tab components
for more information: [https://facebook.github.io/react/docs/context.html](url)